### PR TITLE
Fix Solution to work on ARM

### PR DESCRIPTION
**Pull request by RagingKore**
Fixed: Build on ARM, AnyCPU solution settings was actually using X64

### CODE REVIEW

File: src/Directory.Build.props [hunk 1]
```diff
@@ -10,7 +10,9 @@
 		<PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
 		<Copyright>Copyright 2012-2020 Event Store Ltd</Copyright>
 		<PackageReleaseNotes>https://eventstore.com/blog/</PackageReleaseNotes>
-		<LangVersion>10.0</LangVersion>
+		<TargetFramework>net8.0</TargetFramework>
+		<LangVersion>12.0</LangVersion>
+        <Platforms>AnyCPU;x64;ARM64</Platforms>
```
Comment (timothycoleman) [inline src/Directory.Build.props:15]:
```suggestion
		<Platforms>AnyCPU;x64;ARM64</Platforms>
```

File: src/EventStore.ClusterNode/EventStore.ClusterNode.csproj [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
 	<PropertyGroup>
-		<TargetFramework>net8.0</TargetFramework>
 		<GenerateAssemblyInfo>true</GenerateAssemblyInfo>
 		<OutputType>Exe</OutputType>
 		<ApplicationIcon>app2.ico</ApplicationIcon>
 		<GenerateSupportedRuntime>false</GenerateSupportedRuntime>
-		<Platforms>x64;ARM64</Platforms>
 	</PropertyGroup>
 	<ItemGroup>
+		<PackageReference Include="SharpDotYaml.Extensions.Configuration" Version="0.3.0" />
```
Comment (timothycoleman) [inline src/EventStore.ClusterNode/EventStore.ClusterNode.csproj:9]:
```suggestion
```

File: src/EventStore.Common.Utils/EventStore.Common.Utils.csproj [hunk 3]
```diff
@@ -1,7 +1,4 @@
...
-<Project Sdk="Microsoft.NET.Sdk">
-	<PropertyGroup>
-		<TargetFramework>net8.0</TargetFramework>
-		<Platforms>x64;ARM64</Platforms>
-	</PropertyGroup>
-</Project>
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+    </PropertyGroup>
+</Project>
```
Comment (timothycoleman) [inline src/EventStore.Common.Utils/EventStore.Common.Utils.csproj:4]:
```suggestion
</Project>

```

File: src/EventStore.Core/EventStore.Core.csproj [hunk 4]
```diff
@@ -1,90 +1,87 @@
...
-			<Link>Services\Transport\Grpc\Protos\serverfeatures.proto</Link>
-		</Protobuf>
-		<Protobuf Include="..\Protos\Grpc\redaction.proto" GrpcServices="Server" LinkBase="Services/Transport/Grpc" ProtoRoot="../Protos/Grpc">
-			<Link>Services\Transport\Grpc\Protos\redaction.proto</Link>
-		</Protobuf>
-	</ItemGroup>
-	<ItemGroup>
-		<Folder Include="Services\Transport\Tcp\protos\" />
-	</ItemGroup>
+    <PropertyGroup>
```
Comment (timothycoleman) [inline src/EventStore.Core/EventStore.Core.csproj:2]: accidentally the whitespace in this file

File: src/EventStore.LogV3/EventStore.LogV3.csproj [hunk 5]
```diff
@@ -1,22 +1,19 @@
...
-			<IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
-		</PackageReference>
-	</ItemGroup>
-	<ItemGroup>
-		<ProjectReference Include="..\EventStore.LogCommon\EventStore.LogCommon.csproj" />
-	</ItemGroup>
-	<ItemGroup>
-		<Protobuf Include="**/*.proto" />
-	</ItemGroup>
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (timothycoleman) [inline src/EventStore.LogV3/EventStore.LogV3.csproj:1]: and this one

File: src/EventStore.NETCore.Compatibility/EventStore.NETCore.Compatibility.csproj [hunk 6]
```diff
@@ -1,13 +1,11 @@
...
-		<Platforms>x64;ARM64</Platforms>
-		<GenerateAssemblyInfo>true</GenerateAssemblyInfo>
-	</PropertyGroup>
-	<ItemGroup>
-		<PackageReference Include="System.ServiceModel.Http" Version="6.2.0" />
-		<!-- CVE-2023-29331 -->
-		<PackageReference Include="System.Security.Cryptography.Pkcs" Version="8.0.0" />
-		<PackageReference Include="System.Security.Cryptography.Xml" Version="8.0.0" />
-	</ItemGroup>
+    <PropertyGroup>
```
Comment (timothycoleman) [inline src/EventStore.NETCore.Compatibility/EventStore.NETCore.Compatibility.csproj:2]: ^

File: src/EventStore.TestClient/EventStore.TestClient.csproj [hunk 7]
```diff
@@ -1,39 +1,36 @@
...
-		<ProjectReference Include="..\EventStore.Transport.Http\EventStore.Transport.Http.csproj" />
-		<ProjectReference Include="..\EventStore.Transport.Tcp\EventStore.Transport.Tcp.csproj" />
-	</ItemGroup>
-	<ItemGroup>
-		<None Include="..\EventStore.Common\Log\logconfig.json">
-			<Link>logconfig.json</Link>
-			<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-		</None>
-	</ItemGroup>
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (timothycoleman) [inline src/EventStore.TestClient/EventStore.TestClient.csproj:1]: ^

**Review by timothycoleman — APPROVED** [review]
:+1:

**Review by pvanbuijtene — APPROVED** [review]

**Pull request merged**